libs/base/core.cpp

```diff
@@ -21,23 +21,25 @@ PXT_DEF_STRING(emptyString, "")
 
 static HandlerBinding *handlerBindings;
 
-HandlerBinding *findBinding(int source, int value) {
-    return nextBinding(handlerBindings);
-}
-
 HandlerBinding *nextBinding(HandlerBinding *curr, int source, int value) {
     for (auto p = curr; p; p = p->next) {
-        if ((p->source == source || p->source == DEVICE_ID_ANY) && (p->value == value || p->value == DEVICE_EVT_ANY) {
+        // DEVICE_ID_ANY == DEVICE_EXT_ANY == 0
+        if ((p->source == source || p->source == 0) && 
```

Comment:
why not using the constants?

Reply:
they are not available in in base, which is independent of CODAL.